src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_Format.txt

```diff
@@ -652,7 +652,7 @@ Error({Kind:ErrorKind.InvalidArgument})
 "1,234.57"
 
 >> Text(1234.5678, "[$-pt-BR]#.##0,00", "fr-FR")
-"1 234,57"
+"1 234,57"
```

Comment:
what's the change here?

Reply:
The change is from \u00a0 (no-break space) to \u202f (narrow no-break space). The change is correct (NNBSP is the correct thousands grouping separator for fr-FR). But I wonder how the test is currently passing in the lab (since it let the previous commit go through)

Reply:
This is a Windows 10 / 11 challenge
Now we should be consistent